Documentation/UsingQtCreator.md

```diff
@@ -16,3 +18,28 @@ First, make sure you have a working toolchain and can build and run SerenityOS.
 * Edit the `serenity.includes` file, add the following lines: `.`, `..`, `../..`, `Services/`, `Libraries/`, `Libraries/LibC/`, `Libraries/LibPthread/`, `Libraries/LibM/`, `Toolchain/Local/i686-pc-serenity/include/c++/10.1.0`, `Build/Services/`, `Build/Libraries/`, `AK/`
 
 Qt Creator should be set up correctly now, go ahead and explore the project and try making changes. Have fun! :^)
+
+## Auto-Formatting
+
+You can use `clang-format` to help you with the [style guide](https://github.com/SerenityOS/serenity/blob/master/Documentation/CodingStyle.md). Before you proceed, check that you're actually using clang-format version 10, as some OSes still ship clang-format version 9 by default.
+
+- In QtCreator, go to "Help > About Plugins…"
+- Find the `Beautifier (experimental)` row (for example, by typing `beau` into the search)
+- Put a checkmark into the box
+- Restart QtCreator if it asks you
+- In QtCreator, go to "Tools > Options…"
+- Type "beau" in the search box, go to "Beautifier > Clang Format"
+- Select the "customized" style, click "edit"
+- Paste the entire content of the file `.clang-format` into the "value" box, and click "OK"
+- In the "Beatifier > General" tab, check "Enable auto format on file save"
+- Select the tool "ClangFormat" if not already selected, and click "OK"
+
+Note that not the entire project is clang-format-clean (yet), so sometimes you will see large diffs.
+Use your own judgement whether you want to include such changes. Generally speaking, if it's a few lines then it's a good idea; if it's the entire file then maybe there's a better way to do it, like doing a separate commit, or just ignoring the clang-format changes.
+
+You may want to read up what `git add -p` does (or `git checkout -p`, to undo).
+
+## Compiler Kits
+
+You can slightly improve how well Qt interpretes the code by adding and setting an appropriate "compiler kit".
```

Comment:
interprets*

setting up* (just a matter of taste, I think!)

Reply:
Sure thing – something failed anyway.

Reply:
All done – including the silly shellcheck bug ^^